charabia/src/normalizer/vietnamese.rs

```diff
@@ -0,0 +1,23 @@
+use super::{CharNormalizer, CharOrStr};
+use crate::Token;
+use crate::Script;
+
+pub struct VietnameseNormalizer;
+
+impl CharNormalizer for VietnameseNormalizer {
+    fn normalize_char(&self, c: char) -> Option<CharOrStr> {
+        match c {
+            'Ð' | 'Đ' | 'đ' => Some("d".to_string().into()), // not only Vietnamese, but also many European countries use these letters
```

Comment:
This should say:

```rust
'Ð' | 'Đ' | 'đ' => Some("đ".to_string().into()),
```
since "d" is a different letter, no?

Reply:
I'm not sure about how Slovenian, Croatian, or other countries handle this, but Vietnamese people use a US-layout keyboard with software for typing Unicode, which is manually installed, but not by everyone. 

People would be happy if typing "Da Lat" would produce the same result as "Đà Lạt", "D" as same as "Đ" in digital letters.

This is from Airbnb
![Screenshot 2024-01-18 at 22 42 43](https://github.com/meilisearch/charabia/assets/77522114/f349a7ff-7e4b-43b7-ab4a-10a120e2f2e3)

This is from Skyscanner
![Screenshot 2024-01-18 at 22 46 53](https://github.com/meilisearch/charabia/assets/77522114/061fbd07-a371-4f6d-83f8-1b756a2b1777)

Reply:
Understood, lets keep your implementation then, could you make the CI happy? This way I will be able to merge your PR :)

Reply:
@manythefish Doesn’t the normalizer come before the tokenizer. Given Vietnamese is an n-gram language, I would have thought throwing away the d with stroke Metadata might hurt the n-gram part of the code (downstream). If you normalize it here to just d, don't you also need to wait for all indexed documents to be reindexed for this to work? 

Anyway.... 
I think the real question is can this solution actually meet the users needs.... It's possible it does.

Reply:
Hello @jzabroski  and @ngdbao, the normalizers are processed after the word segmentation, so we already have the token at this step.

Reply:
@ManyTheFish  sounds great now, can't wait to see It get merged :)